lib/SVF-FE/SymbolTableBuilder.cpp

```diff
@@ -206,6 +209,24 @@ void SymbolTableBuilder::collectNullPtrBlackholeSyms(const Value *val)
         symInfo->blackholeSyms.insert(val);
 }
 
+void SymbolTableBuilder::collectArgInNoCallerFunction(const Value *val)
+{
+    if (LLVMUtil::ArgInNoCallerFunction(val))
+        symInfo->getModule()->getArgInNoCallerFunction().insert(val);
+}
+
+void SymbolTableBuilder::collectDeadFunction(const Function * fun)
+{
+    if (LLVMUtil::isDeadFunction(fun))
+        symInfo->getModule()->getIsDeadFunction().insert(fun);
+}
+
+void SymbolTableBuilder::collectReturnInst(const Instruction *inst)
+{
+    if (LLVMUtil::isReturn(inst))
+        symInfo->getModule()->getIsReturn().insert(inst);
```

Comment:
`symInfo->getModule()->addReturnInst()`.

Inside, `addReturnInst()` -- update the set.